src/main/java/seedu/fridgefriend/command/RemoveCommand.java

```diff
@@ -107,5 +89,18 @@ public String getMessagePrintedToUser() {
 
         return message;
     }
+    
+    /**
+     * Checks if the amount of food left for that category is insufficient.
+     * Appends a warning message to the user if true.
+     */
+    private void checkRunningOut() {
+        FoodCategory foodCategory = foodToBeEditted.getCategory();
+        if (fridge.isRunningOut(foodCategory)) {
+            int totalQuantity = fridge.getTotalQuantity(foodCategory);
+            this.runningOutMessage = "\nWARNING! " + foodCategory.toString() + " is running low on food!\nTotal "
```

Comment:
I suggest You are running low on CATEGORY

Reply:
changes made